test/logging/log.py

```diff
@@ -0,0 +1,31 @@
+import logging
+import time
+import os
+# create logger
+logger = logging.getLogger({os.getenv('POD_NAME')})
+logger.setLevel(logging.DEBUG)
+
+# create console handler and set level to debug
+log_path = f"/share/var/log/cortx/{os.getenv('POD_NAME')}"
```

Comment:
again, log destination dir will be read from confStore. Add a note for Cortx services as reference

Reply:
Added comment